include/boost/geometry/io/wkt/write.hpp

```diff
@@ -62,22 +62,63 @@ namespace boost { namespace geometry
 namespace detail { namespace wkt
 {
 
+template <typename Output>
+struct output_formatter
+{
+    template <typename T>
+    static void add(Output & out, T const& v)
+    {
+        out << v;
+    }
+};
+
+template <>
+struct output_formatter<std::string>
+{
+    template <typename T>
+    static void add(std::string & out, T const& val)
```

Comment:
`std::string` needs to be `std::basic_string`

Reply:
And then also take `std::basic_stream` as argument of `to_wkt`.